pkg/proxy/topology.go

```diff
@@ -166,7 +166,7 @@ func canUseTopology(endpoints []Endpoint, svcInfo ServicePort, nodeLabels map[st
 			continue
 		}
 		if endpoint.ZoneHints().Len() == 0 {
-			klog.InfoS("Skipping topology aware endpoint filtering since one or more endpoints is missing a zone hint", "endpoint", endpoint)
+			klog.V(2).InfoS("Skipping topology aware endpoint filtering since one or more endpoints is missing a zone hint", "endpoint", endpoint)
```

Comment:
should we switch all log statements to V(2) ?

Reply:
ping @robscott

Reply:
I think yes, outside of all the startup logs most or all log sites should be `V(N)`

Reply:
do you want to change it?

Reply:
I mean, the two other logs statements in this function, they will have the same issue

Reply:
> do you want to change it?

I didn't want to start combing through all kube-proxy logs sites to update in this PR

> I mean, the two other logs statements in this function, they will have the same issue

I can update those as well, just started with the most annoying, the other 2 combined for ~6M lines/day which is way less than the one I started with. I'll push changes to update the other log statements as well.

Reply:
updated the other 2 locations in this file. If I can find time I'll look into sending a PR for all of kube-proxy, but I can't guarantee that will happen today.

Reply:
Yeah this seems like a good improvement, thanks @bjhaid!